networks/duality-testnet-1/pregenesis/dex.json

```diff
@@ -116,7 +116,7 @@
         }
       ],
       "tokensCount": 28,
-      "feeListList": [
+      "FeeTierList": [
```

Comment:
I've realized now that I missed renaming these properties on the original network genesis file:
https://github.com/duality-labs/duality/blob/ec251371d382332c37006693fd5f835b1a6fbe26/networks/duality-testnet-1/genesis.json#L106-L107

Inside a Docker startup process these lines get overridden by the changes in this PR so they don't actually affect the setup, but they should have been updated anyway.